Running in wrapper mode for '10'

Running python job.

[name: "CPU-QUOTA"
int_val: 60
int_val: 27
int_val: 21
int_val: 67
int_val: 18
int_val: 87
int_val: 13
, name: "DISK"
int_val: 23
int_val: 28
int_val: 35
int_val: 7
int_val: 6
int_val: 8
int_val: 12
, name: "MEMORY"
int_val: 38
int_val: 29
int_val: 23
int_val: 11
int_val: 32
int_val: 20
int_val: 31
, name: "NET"
int_val: 40
int_val: 11
int_val: 10
int_val: 36
int_val: 26
int_val: 26
int_val: 5
]
The paramters are {u'CPU-QUOTA': array([60, 27, 21, 67, 18, 87, 13]), u'NET': array([40, 11, 10, 36, 26, 26,  5]), u'DISK': array([23, 28, 35,  7,  6,  8, 12]), u'MEMORY': array([38, 29, 23, 11, 32, 20, 31])}
{'13.56.14.158': ['haproxy', 'node-app', 'mongo'], '18.144.43.31': ['node-app', 'mongo']}
----------------------------------------
Problem running the job:
(<class 'paramiko.ssh_exception.BadAuthenticationType'>, BadAuthenticationType('Bad authentication type', [u'publickey']), <traceback object at 0x7fd41b7b71b8>)
Traceback (most recent call last):
  File "/home/ubuntu/throttlebot/src/spearmint/spearmint/runner.py", line 42, in job_runner
    elif job.language == PYTHON: run_python_job(job)
  File "/home/ubuntu/throttlebot/src/spearmint/spearmint/runner.py", line 142, in run_python_job
    result = module.main(job.id, params)
  File "/home/ubuntu/throttlebot/src/spearmint/bayOptSearch/bayOpt.py", line 104, in main
    return explore_spearmint(workload_config, params)
  File "/home/ubuntu/throttlebot/src/spearmint/bayOptSearch/bayOpt.py", line 147, in explore_spearmint
    client = re.get_client(machine)
  File "/home/ubuntu/throttlebot/src/remote_execution.py", line 6, in get_client
    client.connect(ip, username="quilt", password="")
  File "/home/ubuntu/.local/lib/python2.7/site-packages/paramiko/client.py", line 424, in connect
    passphrase,
  File "/home/ubuntu/.local/lib/python2.7/site-packages/paramiko/client.py", line 714, in _auth
    raise saved_exception
BadAuthenticationType: ('Bad authentication type', [u'publickey']) (allowed_types=[u'publickey'])
None
----------------------------------------
Job file reloaded.
Could not find value in output file.
Job failed in 0.44 seconds.
